tools/FlatFlowMigrationCli/MigrationLogger.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.DotNet.ProductConstructionService.Client.Models;
+using Microsoft.Extensions.Logging;
+
+namespace FlatFlowMigrationCli;
+
+/// <summary>
+/// Class that logs the operations in console and into a file instead of performing them.
+/// </summary>
+internal class MigrationLogger : ISubscriptionMigrator
+{
+    private readonly ILogger<MigrationLogger> _logger;
+    private readonly string _outputPath;
+
+    public MigrationLogger(ILogger<MigrationLogger> logger, string outputPath)
+    {
+        _logger = logger;
+        _outputPath = outputPath;
+    }
+
+    public async Task DisableSubscriptionAsync(Subscription subscription)
+    {
+        _logger.LogInformation("Would disable a subscription {sourceRepository} - {targetRepository} / {subscriptionId}",
```

Comment:
```suggestion
        _logger.LogInformation("Would disable a subscription {sourceRepository} -> {targetRepository} / {subscriptionId}",
```
nit: subscriptions are "directional" so I think an arrow closer describes them